scheduler/src/cook/util2.clj

```diff
@@ -13,16 +13,16 @@
 ;; See the License for the specific language governing permissions and
 ;; limitations under the License.
 ;;
-(ns cook.mesos.util
+(ns cook.util2
```

Comment:
This I don't like...can we roll it into `cook.util`? Or maybe `cook.scheduler.util`?

Reply:
After discussion we renamed this as cook.tools